translations/es.ts

```diff
@@ -1200,7 +1200,7 @@ en lugar de la barra de tareas del sistema.</translation>
 will be sent to them when they appear online to you.</source>
         <comment>toolTip for Faux offline messaging setting</comment>
         <translation>Los mensajes que intentes enviar a tus amigos cuando estén desconectados
-serán enviados cuando los veas aparecer de nuevo en línea.</translation>
+serán enviados cuando ellos vuelvan a conectarse.</translation>
```

Comment:
@kehugter I hope you like this version more. I think it's clearly an improvement, as much for clarity and correctness, as for brevity.